src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1506,10 +1518,9 @@ export default class AircraftModel {
             }
 
             case FLIGHT_PHASE.WAITING:
-                const iAmTheNextDeparture = this.fms.departureRunwayModel.isAircraftNextInQueue(this.id);
-
-                if (this.shouldTakeOffWhenRunwayIsClear && iAmTheNextDeparture) {
-                    const lastDeparture = this.fms.departureRunwayModel.lastDepartedAircraftModel;
+                const runway = this.fms.departureRunwayModel;
+                if (isAutoTower && runway.isAircraftNextInQueue(this.id)) {
+                    const lastDeparture = runway.lastDepartedAircraftModel;
                     const iAmTheFirstEverDeparture = lastDeparture === null;
```

Comment:
I know this was here before, but could we rename `iAmTheFirstEverDeparture` to `isFirstEverDeparture`?